tests/unit/amo/components/TestSearchForm.js

```diff
@@ -109,17 +135,20 @@ describe(__filename, () => {
       const wrapper = mountComponent();
 
       sinon.assert.notCalled(router.push);
-      wrapper.find('input').simulate('change', createFakeChangeEvent('adblock'));
```

Comment:
I see a lot of line shortening in here which creates noise during the review process. How about we switch to 80 char line length in the lint rules? I personally prefer it too as it helps me use split view more efficiently in my editor. /cc @willdurand

Reply:
I'm cool with that. I'll remove it from this review though, sorry about that. I just can't unsee it going over 80 and I wanna fix it, but you're right. I should resist 😉

Reply:
👍